integration_tests/src/main/python/data_gen.py

```diff
@@ -826,6 +826,19 @@ def gen_scalars_for_sql(data_gen, count, seed=0, force_no_nulls=False):
            decimal_gen_default, decimal_gen_scale_precision, decimal_gen_same_scale_precision,
            decimal_gen_64bit]
 
+common_cudf_types_gen = all_basic_gens[:-1]
```

Comment:
This is very fragile, especially since it's so far removed from where `all_basic_gens` is defined.  I'd much rather see `all_basic_gens` be based on `common_cudf_types_gen` rather than assuming we know how to slice up `all_basic_gens` accordingly.  The next time someone touches `all_basic_gens` this is almost guaranteed to break.

Reply:
Thanks for reivew. Good catch. Updated.